security-analysis/security-analysis-api/src/main/java/com/powsybl/security/SecurityContingencyStatus.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security;
+
+/**
+ * @author Bertrand Rix <bertrand.rix at artelys.com>
+ */
+public enum SecurityContingencyStatus {
```

Comment:
PostContingencyComputationStatus would be more clear. 
And yes ComponentResult.Status would have better been named ComponentResult.ComputationStatus... (could be done later)